src/dotnet/commands/dotnet-build/Program.cs

```diff
@@ -15,13 +15,19 @@ namespace Microsoft.DotNet.Tools.Build
 {
     public class BuildCommand
     {
-        public static int Run(string[] args)
+        public static int Run(string[] args) => Run(args, null);
+
+        public static int Run(string[] args, WorkspaceContext workspace)
```

Comment:
I agree that this immediate PR does not create a disadvantage. It does, however introduce a pattern which then leads to bifurcation. We're all acknowledging that this is a quick fix. We need to be diligent about preventing the pattern from growing and about developing a solid cross-platform solution.